src/Form.js

```diff
@@ -1,19 +1,331 @@
+//             obj[Number(number)] += 1;
+//           }
+//         });
+        
+//         // return object key with highest value.
+//         let highestValue = 0;
+//         let highestValueKey = -Infinity;
+      
+//         for (let key in obj) {
+//           const value = obj[key];
+//           if (value > highestValue) {
+//             highestValue = value;
+//             highestValueKey = key;
+//           }
+//         }
+      
+//         // convert key back to number
+//         setmodeNumber(Number(highestValueKey));
+//       }
+//       event.preventDefault();
+// //   }
+// const handleEvent = event=>{
+//   invalid=""
+// let e=event.target.value
+// if(event.target.values==="sum"){
+// let initial=0
+// for(let i=0; i<e.length;i++){
+//   initial+= Number(e[i])
+// }
+// setsumNumber(`${initial}`)
+
+// }
+// else if(event.target.values==="average"){
+//   let initial=0
+// for(let i=0; i<e.length;i++){
+//   initial+= Number(e[i])
+// }
+// setAvgNumber(`${initial/e.length}`);
+// }
+
+// else if(event.target.values==="mode"){
+//   const obj = {};
+//   // loop over array
+//   e.forEach(number => {
+//     // for each number in array,
+//     // if it doesn't already exist as a key on the
+//     // object, create one and set its value to 1.
+//     if (!obj[Number(number)]) {
+//       obj[Number(number)] = 1;
+//     } else {
+//       // if it already exists as key on the object,
+//       // increment its corresponding value.
+//       obj[Number(number)] += 1;
+//     }
+//   });
+  
+//   // return object key with highest value.
+//   let highestValue = 0;
+//   let highestValueKey = -Infinity;
+
+//   for (let key in obj) {
+//     const value = obj[key];
+//     if (value > highestValue) {
+//       highestValue = value;
+//       highestValueKey = key;
+//     }
+//   }
+
+  // convert key back to number
+//   setmodeNumber(Number(highestValueKey));
+// }
+// else{invalid = "Invalid"}
+// event.preventDefault();
+// }
+// let valueArray=[]
+const handleSubmit = event => {
+  document.getElementById("render").innerHTML=""
+  event.preventDefault();
+
+
+  document.getElementById("render").innerHTML = `${details.val}`
+// if(details.val !== NaN){
+//   document.getElementById("values").value= ""
+// }
+
+  if(details.values.length===0 || details.operation === "" || details.val === NaN){
+    document.getElementById("render").innerHTML = "Invalid input."
+  }
+  // Number(document.getElementById("render").innerHTML) === NaN
+  console.log((document.querySelector("form input").value.split(",")).map(element => !isNaN(element)).includes(true));
+if((document.querySelector("form input").value.split(",")).map(element => !isNaN(element)).includes(true)){
+  document.querySelector("form input").value = "";
+  document.querySelector("form input").classList.add("error")
+  document.querySelector("form select").value = ""
+  document.querySelector("form select").classList.add("error")
+}
+}
```

Comment:
There are two easier more common ways to reset a form.

**The Vanilla JS way** - `event.target.reset()` - This way should not be used when we're using React, because all the inputs state is controlled by react state
**The React.js way** - 
`setDetails({
  values:[],
  operation:"",
  val:0
  })`
  This is the right way to reset a form in React. You reset the state that controls the inputs